topic04-git-introduction/book-git-tutorial/01.Basics.md

```diff
@@ -0,0 +1,46 @@
+---

```

Comment:
@maciejmarchel12  Do we need these three dashes here? Also lets be careful with the spacing. We are using this markdown to build html files so not sure if the spaces will render in the html. Lets keep the spacing neat just in case.

Reply:
@laurafitzgerald The three dashes create a line and it makes the document look neater.

Reply:
@maciejmarchel12 ok if that's the case let's get agreement from the team and use it across all the pages. @AdamLalor @dominikkawka do you guys agree with this and if so lets add it.